### ISSUE TITLE
Not possible to add Workflow without depencency

### ISSUE THREAD

**Issue by viezel** [ksassnowski/venture#20]
its documented [here](https://laravel-venture.netlify.app/usage/nesting-workflows.html#adding-a-nested-workflow) that its possible to add a workflow without dependencies. However, this is not the case. 

```php
->addWorkflow(new FirstOrderSigning($this->document)),
```

```
Too few arguments to function Sassnowski\Venture\WorkflowDefinition::addWorkflow(), 1 passed .. and exactly 2 expected 
```

**Comment by ksassnowski** [comment]
Looks like I forgot to make the parameter optional for this method. Will fix this in the next patch release.

**Comment by ksassnowski** [comment]
This was merged into master (86ba1970c52b42bb177da5133783ce2b74441cd6) and will be part of the 2.1.0 release.
